client/scss/settings/_variables.scss

```diff
@@ -64,6 +64,7 @@ $color-input-error-bg: lighten(saturate($color-red, 28), 45);
 
 $color-button: $color-teal;
 $color-button-hover: $color-teal-darker;
+$color-button-active: $color-teal-dark;
```

Comment:
as mentioned above, leveraging the .secondary convention that already exists gets us to an alternate style without adding a new convention of this kind of 'active' button.